parts/hasonlofejl.tex

```diff
@@ -0,0 +1,46 @@
+%2. fejezet
+
+%2.1
+\section{A videójátékok története}
+\label{resz2_1}
+
+% \Picture{A társasjáték}{2/1}{width=10cm}
+
+Videójátékoknak nevezzük azokat a típusú játékokat, ahol a játékosok egy felhasználói felületen keresztüllépnek interakcióba a játékkal. Története az 1950-es évekre vezethető vissza, ebben az évtizedben kezdte el foglalkoztatni az embereket az az ötlet, hogy a szórakozás élményét elektromos eszközök segítségével érjék el. Az első videojáték egy asztalitenisz szimulátor volt, 1958-ban készítette el William Higinbotham és a „Tennis for Two” nevet viselte. A játék egy asztali analóg számítógépre készült el, és egy oszcilloszkópot használt a megjelenítésre.
+
+\section{Sportjátékok}
+\label{resz2_2}
+
+Napjainkban nagy népszerűségnek örvendenek az olyan játékok, ahol a játékosok interakcióba tudnak lépni egymással, ez lehet együttműködés vagy egymás elleni versengés is. Mivel a versenyszellem velünk született tulajdonságunk, ezért a felhasználók minden tőlük telhetőt megtesznek azért, hogy a ranglista élére kerülhessenek. Erre alapozva sok játékkészítő nagyon kreatívan határozta meg a fejlődés árát. Az alábbiakban pár olyan játékot mutatok be, amelyeknél ez az ár a testmozgás.
+
+\section*{Zombies, Run!}
+\label{resz2_2_1}
```

Comment:
labeleknek erdemes ertelmes nevet adni. itt pl "zombiesrun" vagy valami. Ha kesobb ezt valahova athelyezzuk, furan is veszi ki magat a 2_2_1, meg nem is logikus igy. Ez a tobbi hasonlo helyre is vonatkozik. A label lenyege pont az, hogy ne kelljen azzal foglalkozni, hova kerul a tartalom, attol fuggetlenul tudjunk ra hivatkozni.